sig-release/20220831.md

```diff
@@ -0,0 +1,49 @@
+SIG RELEASE
+
+Beware: the contents of this pad will be publically available and the resulting minutes stored on github:
```

Comment:
Do we want to drop this from the final minutes?

Reply:
thanks @garloff!